css/styles.css

```diff
@@ -0,0 +1,301 @@
+html, body{
+    background-color: #FFFFFF;
+    font-family: Calibri, Tahoma, Geneva, Verdana, sans-serif;
+}
+
+header{
+    background-color: aqua;
+    position: absolute;
+    width: 1920px;
+    height: 840px;
+    left: 0px;
+    top: 0px;
+}
+
+.JK{
+    position: absolute;
+    width: 556px;
+    height: 109px;
+    left: 682px;
+    top: 296px;
+    font-family: Calibri;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 48px;
+    line-height: 64px;
+    text-align: center;
+    color: #D4C17F;
+}
+
+header.inner{
+    width: 1080px;
+    margin: 0 auto;
+    position: relative;
+    height: 400px;
+}
+
+.nav{
+    position: absolute;
+    top: 20px;
+    right: 120px;
+}
+
+.nav-list{
+    list-style-type: none; 
+}
+
+.nav-list li{
+    float: left;
+    margin: 0 35px 0 0;
+}
+
+.nav_list li:last-child{
+    margin-right: 0;
+}
+
+.navigation{
+    position: relative;
+    width: 800px;
+    height: 50px;
+    left: 692px;
+    top: 745px;
+    display: flex;
+    font-family: Raleway;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    color: #FFFFFF;
+}
+
+.number{
+    position: absolute;
+    width: 139px;
+    height: 21px;
+    left: 1046px;
+    top: 745px;
+    font-family: Raleway;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    color: #FFFFFF;
+}
+
+.about_us{
+    background-color: blueviolet;
+    position: absolute;
+    width: 794px;
+    height: 180px;
+    left: 563px;
+    top: 960px;
+}
+
+.historical_parks{
+    display: flex;
+    width: 193px;
+    height: 47px;
+    left: 563px;
+    top: 1093px;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: 300;
```

Comment:
Use word-values for the rule, because most of fonts has thickness equal to word notation, but has no intermediate thickness like 300 or 500. [Read comments here](http://htmlbook.ru/css/font-weight).